clcache.py

```diff
@@ -57,6 +57,16 @@
 
 HashAlgorithm = hashlib.md5
 
+
+# The codec that is used by clcache to store compiler STDOUR and STDERR in
+# output.txt and stderr.txt.
+# This codec is up to us and only used for clcache internal storage.
+# For possible values see https://docs.python.org/2/library/codecs.html
+CACHE_COMPILER_OUTPUT_STORAGE_CODEC = 'mbcs'
```

Comment:
If "mbcs" is a system-dependant encoding, wouldn't it be better to use UTF-8 for the cache such that it can safely be shared between multiple systems with potentially different encodings?